radix-engine-tests/tests/application/metering.rs

```diff
@@ -67,6 +67,11 @@ fn run_all(mode: CostingTaskMode) {
             &run_mint_small_size_nfts_from_manifest,
             "cost_mint_small_size_nfts_from_manifest.csv",
         );
+
+        // Run cost tests for crypto_utils test from Cuttlefish onward
+        if protocol_version >= ProtocolVersion::Cuttlefish {
```

Comment:
Isn't this condition always going to be true from this point onwards?

Reply:
No, because we iterate over versions starting from `Genesis`
see [line 37](https://github.com/radixdlt/radixdlt-scrypto/pull/1952/files/5a97d1350e181d2359eb514bb146f66799d0dccd#diff-9f9d7f5a18faa9e6112ad24f3c3f6cb17a017e88b40e3fb62ac3d5268dc027c3R37)